openshift-fluentd-syslog-journal.conf

```diff
@@ -0,0 +1,25 @@
+<source>
+  @type systemd
+  path "#{ENV['DATA_DIR']}/journal"
+  pos_file "#{ENV['DATA_DIR']}/journal.pos"
+  tag system.var.log.messages
```

Comment:
but the filters applied to the source are different. i.e. the very reason why the tags should be different.